lte/gateway/c/session_manager/SessionState.h

```diff
@@ -163,7 +164,13 @@ class SessionState {
 
   void insert_pdr(SetGroupPDR* rule);
 
-  void set_remove_all_pdrs();
+  void set_all_pdrs(enum PdrState);
+
+  bool search_pdr(unsigned int id);
+
+  void reset_throttle();
+
+  bool inc_throttle();
 
   void insert_far(SetGroupFAR* rule);
```

Comment:
please include comments about the usage of these methods.

Reply:
Fixed, udpate in .h and .cpp as well